### PR TITLE
Added requirements for merging features

### PR DESCRIPTION
**Pull request by SeqLaz**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3762 Merging conditions](https://app.clickup.com/t/2192114/QF-3762)

File: documentation/how-to/digitize.en.md [hunk 1]
```diff
@@ -145,6 +145,19 @@ QField allows you to merge features and their geometries into a single feature.
 features on the map, select them in the features list and merge them by selecting
 the *Merge Selected Features* in the *Menu (⁝) Button*.
 
+To execute polygon merging within QField, ensure the following conditions:
```
Comment (nirvn) [inline documentation/how-to/digitize.en.md:148]: It's not only polygon merging, it's all geometry types

Comment (SeqLaz) [reply documentation/how-to/digitize.en.md:148]: Great, I will fix it

File: documentation/how-to/digitize.en.md [hunk 2]
```diff
@@ -145,6 +145,19 @@ QField allows you to merge features and their geometries into a single feature.
 features on the map, select them in the features list and merge them by selecting
 the *Merge Selected Features* in the *Menu (⁝) Button*.
 
+To execute merging within QField, ensure the following conditions:
+
+1. **Editable Vector Layer:**
+   - The target layer must an editable layer.
```
Comment (m-kuhn) [inline documentation/how-to/digitize.en.md:151]:
```suggestion
   - The target layer must be an editable layer.
```

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**